renku/core/models/projects.py

```diff
@@ -88,8 +98,52 @@ def __attrs_post_init__(self):
                 # this assumes the project is being newly created
                 self.creator = Creator.from_git(self.client.repo)
 
-        if not self._id and self.client:
-            self._id = self.client.project_id
+        if not self._id or self._id.startswith('file://'):
+            self._id = self.project_id
+
+    @property
+    def project_id(self):
+        """Return the id for the project based on the repo origin remote."""
+        from renku.core.models.git import GitURL
+
+        # RENKU_DOMAIN environment variable is the host override
```

Comment:
I think we should create and .env-example file as an example .env file where we keep all possible configuration variables. That way we have a simple and easier way on how renku can be tweaked. Of course, in the long run this should be all documented properly.

Reply:
Usually, .env is in package root so I'm okay with having .env-example in project root as well.

Reply:
ok this should then be done as part of the templating work and project-level settings - see also #730